cpp/src/io/orc/reader_impl.cu

```diff
@@ -944,7 +944,15 @@ table_with_metadata reader::impl::read(size_type skip_rows,
       auto col_type = to_type_id(
         _metadata.get_col_type(col.id), _use_np_dtypes, _timestamp_type.id(), decimal_as_float64);
       CUDF_EXPECTS(col_type != type_id::EMPTY, "Unknown type");
+<<<<<<< HEAD
```

Comment:
This merge conflict has not been resolved

Reply:
Good catch. Fixed.